tests/cpu-vm

```diff
@@ -79,16 +73,35 @@ for i in $(seq "${cpuCount}" -1 1); do
   [ "$(lxc exec v1 -- ls /sys/devices/system/cpu | grep -xEc 'cpu[[:digit:]]+')" -eq "${i}" ]
 done
 
-# Try doing pinning while VM is running (shouldn't work)
-! lxc config set v1 limits.cpu=1,2 || false
+echo "==> Check that there is no CPU pinning set"
+QEMU_PID=$(lxc info v1 | awk '/^PID:/ {print $2}')
+! taskset --cpu-list -a -p "${QEMU_PID}" | grep -E ':\s+[0-9]+$' || false
```

Comment:
This currently fails on `latest/edge` as it doesn't have the `limits_cpu_pin_strategy` extension but the old autopinning by default.

The autopinning goes away and never landed in a LTS so maybe that's an acceptable situation until https://github.com/canonical/lxd/pull/14171 is merged?

That said, I'd still like a full run on `latest/edge` so maybe stub out that bit for a quick CI run?

Reply:
Since the auto pinning will be disabled by default on `latest/edge` with the merge of https://github.com/canonical/lxd/pull/14171, I think we're covered. I've done a full run of `cpu-vm` against `latest/edge` with https://github.com/canonical/lxd/pull/14171's LXD side-loaded and it passed ✅.

Reply:
Works for me, thanks for confirming!